ui/components/multichain/asset-picker-amount/asset-picker/index.scss

```diff
@@ -0,0 +1,19 @@
+.asset-picker {
+  display: flex;
+  align-items: center;
+  padding: 7px;
```

Comment:
You can use the style utility props included in all component library components instead of having to write more CSS. This also has some mild performance benefits because it reduces the CSS that is loaded with the application